Firestore/Source/Model/FSTMutationBatch.mm

```diff
@@ -178,14 +182,14 @@ + (instancetype)resultWithBatch:(FSTMutationBatch *)batch
       version = commitVersion;
     }
 
-    docVersions = [docVersions dictionaryBySettingObject:version.value() forKey:mutations[i].key];
+    docVersions.emplace(mutations[i].key, version.value());
```

Comment:
Is it possible for two mutations to have the same key? If so, I think this is slightly different than the old code. (Last wins with the old code, first wins with the new.)

Reply:
Two mutations can have the same key. Great call.

I've changed to using `operator[] =` which has semantics consistent with the old code.